packages/rules/src/url.ts

```diff
@@ -0,0 +1,26 @@
+import { getSingleParam } from './utils';
+
+const urlValidator = (
+  value: unknown,
+  params: [string | RegExp | undefined] | { pattern?: string | RegExp }
+): value is string => {
+  if (typeof value !== 'string') {
+    return false;
+  }
```

Comment:
Minor comment, all rules that are not `required` do allow falsy values like `null`, `undefined`, and empty strings to avoid making the rule behave as required.

```suggestion
import { getSingleParam, isEmpty } from './utils';

const urlValidator = (
  value: unknown,
  params: [string | RegExp | undefined] | { pattern?: string | RegExp }
): value is string => {
  if (isEmpty(value)) {
    return true;
  }
```

Reply:
Sure thing! Changed it.